src/lib/api.js

```diff
@@ -68,6 +68,21 @@ function api(options, callback) {
       return callback(err);
     }
 
+    // 'err' is used to signal library errors, not network errors,
+    // so we also need ot make sure to check what the network status
+    // code was, before we can assume everything worked fine:
+    if (res.statusCode !== 200 && callback) {
```

Comment:
Should we instead check to see if the status code is 4xx or 5xx [like superagent does](http://visionmedia.github.io/superagent/#error-handling)?

At the very least, my concern here is that non-200 [2xx success codes](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#2xx_Success) like `201 Created` will be treated as errors here, which seems like it might become a problem if the webmaker API ever starts returning those kinds of codes.